<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RechercheContactWindow</class>
 <widget class="QWidget" name="RechercheContactWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chercher un contact</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>15</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Chercher un contact par :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="rechercheContactComboBox"/>
       </item>
       <item>
        <widget class="QLineEdit" name="rechercheLineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="placeholderText">
          <string>Nom ou Entreprise</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Créé entre le</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEditDebut">
         <property name="dateTime">
          <datetime>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2021</year>
           <month>1</month>
           <day>1</day>
          </datetime>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>et le</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateEditFin">
         <property name="dateTime">
          <datetime>
           <hour>0</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2021</year>
           <month>1</month>
           <day>1</day>
          </datetime>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Trier par :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="triListeComboBox"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="quitButton">
     <property name="text">
      <string>Quitter</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QListView" name="listeContacts">
     <property name="cursor" stdset="0">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
